From 6faf01a897bbfc0e9e21835e5facdc8a4fd05b6a Mon Sep 17 00:00:00 2001
From: "cherry.chen" <cherry.chen@rock-chips.com>
Date: Mon, 20 Aug 2018 15:14:05 +0800
Subject: [PATCH] arm: configs: enable aloop driver for rk3308 aarch32

Because of resample, some speaker processing(such as EQ/DRC)
cannot be done after dmix in alsa. Enable aloop driver make them
can be handled between virtual hardware to the real hardware.

Change-Id: I05b92632359fabd5c1dd8b051295ad4d17dddc8c
Signed-off-by: Cherry Chen <cherry.chen@rock-chips.com>
---
 arch/arm/configs/rk3308_linux_aarch32_debug_defconfig | 3 +--
 arch/arm/configs/rk3308_linux_aarch32_defconfig       | 2 +-
 2 files changed, 2 insertions(+), 3 deletions(-)

diff --git a/arch/arm/configs/rk3308_linux_aarch32_debug_defconfig b/arch/arm/configs/rk3308_linux_aarch32_debug_defconfig
index 820a518ae399..186a224d77d6 100644
--- a/arch/arm/configs/rk3308_linux_aarch32_debug_defconfig
+++ b/arch/arm/configs/rk3308_linux_aarch32_debug_defconfig
@@ -137,7 +137,6 @@ CONFIG_DEVFREQ_THERMAL=y
 CONFIG_ROCKCHIP_THERMAL=y
 CONFIG_WATCHDOG=y
 CONFIG_DW_WATCHDOG=y
-CONFIG_MFD_CORE=y
 CONFIG_MFD_RK808=y
 CONFIG_REGULATOR_FIXED_VOLTAGE=y
 CONFIG_REGULATOR_GPIO=y
@@ -147,7 +146,7 @@ CONFIG_SOUND=y
 CONFIG_SND=y
 CONFIG_SND_HRTIMER=y
 # CONFIG_SND_SUPPORT_OLD_API is not set
-# CONFIG_SND_DRIVERS is not set
+CONFIG_SND_ALOOP=y
 # CONFIG_SND_SPI is not set
 CONFIG_SND_SOC=y
 CONFIG_SND_SOC_ROCKCHIP=y
diff --git a/arch/arm/configs/rk3308_linux_aarch32_defconfig b/arch/arm/configs/rk3308_linux_aarch32_defconfig
index 0ea6147348a8..b480660d23be 100644
--- a/arch/arm/configs/rk3308_linux_aarch32_defconfig
+++ b/arch/arm/configs/rk3308_linux_aarch32_defconfig
@@ -142,7 +142,7 @@ CONFIG_SOUND=y
 CONFIG_SND=y
 CONFIG_SND_HRTIMER=y
 # CONFIG_SND_SUPPORT_OLD_API is not set
-# CONFIG_SND_DRIVERS is not set
+CONFIG_SND_ALOOP=y
 # CONFIG_SND_SPI is not set
 CONFIG_SND_SOC=y
 CONFIG_SND_SOC_ROCKCHIP=y
-- 
2.35.3

